entries/jQuery.trim.xml

```diff
@@ -10,7 +10,10 @@
   <desc>Remove the whitespace from the beginning and end of a string.</desc>
   <longdesc>
     <div class="warning">
-      <p>Note: This API has been deprecated in jQuery 3.5; please use the native <a href="https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/trim"><code>String.prototype.trim</code></a> method instead.</p>
+      <p>Note: This API has been deprecated in jQuery 3.5; please use the native <a href="https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/trim"><code>String.prototype.trim</code></a> method instead.
+      Unlike <code>jQuery.trim</code>, <code>String.prototype.trim</code> does not work with types other than strings (<code>null</code>, <code>undefined</code>, <code>Number</code>). 
+      Make sure that your code is compatible when migrating.
+      </p>
```

Comment:
Do you want that to be two paragraphs? It makes sense but then you'll need to insert those markers

Reply:
No, I assumed it to be a single paragraph.